testing/test_step_saving.py

```diff
@@ -206,15 +206,15 @@ def given_saved_pipeline(tmpdir):
     given_saved_some_step(multiply_by=4, name=SOME_STEP_2, path=create_some_step2_path(tmpdir, True))
     given_saved_some_step(multiply_by=6, name=SOME_STEP_3, path=create_some_step3_path(tmpdir, True))
 
-    pickle_checkpoint_step = PickleCheckpointStep(cache_folder=tmpdir)
-    pickle_checkpoint_step.name = CHECKPOINT
-    dump(pickle_checkpoint_step, create_some_checkpoint_path(tmpdir, True))
+    checkpoint = MiniCheckpoint()
+    checkpoint.name = CHECKPOINT
+    dump(checkpoint, create_some_checkpoint_path(tmpdir, True))
 
     p = ResumablePipeline([
         (SOME_STEP_1, MultiplyBy(multiply_by=1)),
         (PIPELINE_2, ResumablePipeline([
             (SOME_STEP_2, MultiplyBy(multiply_by=1)),
-            (CHECKPOINT, PickleCheckpointStep(cache_folder=tmpdir)),
+            (CHECKPOINT, MiniCheckpoint()),
```

Comment:
Can you Really put minicheckpoints there?

Reply:
renamed to default checkpoint